site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -77,23 +116,23 @@ the page that the content script has been [injected][content-inject] into.
 }
 ```
 
-### Getting background data into a content script {: #example-content-msg }
+### Send data from the service worker to a content script {: #example-content-msg }
 
 Its common for an extension's content scripts to need data managed by another part of the extension,
-like the extension's background script. Much like two browser windows opened to the same web page,
-these two contexts cannot directly access each other's values. Instead, the extension can use
-[message passing][message-passing] to coordinate across these different contexts.
+like the service worker. Much like two browser windows opened to the same web page, these
+two contexts cannot directly access each other's values. Instead, the extension can use [message
+passing][doc-messages] to coordinate across these different contexts.
 
-In this example, the content script needs some data from the extension's background script in order
-to initialize its UI. To get this data, it passes a `get-user-data` message to the background, and
-the background responds with a copy of the user's information.
+In this example, the content script needs some data from the extension's service worker to
```

Comment:
Found all mentions of background scripts or background pages and replaces with service worker.

Reply:
Thanks for doing this!